File: Bucles/Ejercicio7-Bucles.cpp
/*7. Escriba un programa que calcule el valor de :
1+2+3+....+n*/
#include<iostream>
#include<stdlib.h> //La librería conio.h está obsoleta
using namespace std;
int main(){
    int n, sumatoria = 0;
    cout<<"Digite valor de n: ";
    cin>>n;
    for(int i=1; i<=n; i++){
        sumatoria = sumatoria + i;
    }
    cout<<"\nLa sumatoria es: "<<sumatoria<<endl;
    system("pause");
    return 0;
}

File: Comparativa Python/Ejercicio7-Bucles.py
"""7. Escriba un programa que calcule el valor de :
1+2+3+....+n"""

n = int(input('Digite número de iteraciones: '))

sumatoria = 0
for i in range(1, n+1, 1):
    sumatoria = sumatoria + i

print('El resultado de la sumatoria es: ', str(sumatoria))
